test/e2e-test/appconfig_dependency_test.go

```diff
@@ -0,0 +1,475 @@
+package controllers_test
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/crossplane/crossplane-runtime/apis/core/v1alpha1"
+
+	"github.com/crossplane/oam-kubernetes-runtime/apis/core/v1alpha2"
+	"github.com/crossplane/oam-kubernetes-runtime/pkg/oam/util"
```

Comment:
AFAICT, this is already gofmt-ed. The CI also has checks for that. What are you expecting to change here?

Reply:
There should not be 5 import groups

Reply:
> There should not be 5 import groups

What's the reason?

Reply:
I have tried hard with the code checks before so I can guess this is what @My-pleasure tried:

- go built-in should be together in one block
- ginkgo are imported as `. xxx` so they are in one block
- external deps are in one block
- `crossplane/crossplane-runtime` are in one block otherwise the CI will fail
- `crossplane/oam-kubernetes-runtime/` in one block otherwise the CI will fail

Reply:
This seems valid to me as dealing with the import rules from the CI would lead to this result..

Reply:
Have you tried to use `goimports`, it's the same idea as `gofmt` so we don't need to talk about imports in reviews.

Reply:
I just tried to checkout the code and run goimports. It didn't print out any diff so it concludes that this is correctly formatted.